src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.figures;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(String color, String name, int length, int width) {
+        super(color, name = "Rectangle");
```

Comment:
In the constructor, the assignment `name = "Rectangle"` is unnecessary since `name` is being set through the `super` call. You should remove `name = "Rectangle"` and just call `super(color, "Rectangle")`.